app/client/components/consent/PrivacySettings.tsx

```diff
@@ -33,11 +32,15 @@ const PURPOSES_ID = "purposes";
 const SCROLLABLE_ID = "scrollable";
 const HEADER_ID = "header";
 
-const isProd = conf.DOMAIN === "theguardian.com";
+let domain: string;
```

Comment:
Be easy to write this as a ternary operator to avoid the mutability, but this is a *super* minor point, no stressing at all.